src/Database/LSMTree/Internal/Run.hs

```diff
@@ -196,11 +197,12 @@ fromMutable fs hbio caching refCount builder = do
 fromWriteBuffer :: HasFS IO h
                 -> HasBlockIO IO h
                 -> RunDataCaching
+                -> RunBloomFilterAlloc
```

Comment:
We could bundle these two arguments (and in the future potentially more things) into a single run creation config/setting type (also in `Merge.new`, `mergeRuns`), but maybe it's best to wait and only do it once we really add a third thing.